source/SH3/arc/vfile.cpp

```diff
@@ -31,7 +33,7 @@ bool sh3_arc_vfile::Open(sh3_arc& mft, const std::string& fname)
         return open;
     }
 
-    return open;
+    return false;
```

Comment:
You can avoid indentation of the previous code with an early return (`if(open) return false;`).